plugins/optimization-detective/readme.txt

```diff
@@ -1,4 +1,4 @@
-=== Optimization Detective ===
+=== Optimization Detective (Developer Preview) ===
 
 Contributors:      wordpressdotorg
 Requires at least: 6.3
```

Comment:
```suggestion
Requires at least: 6.4
```

Should be 6.4

Reply:
Oh, I missed that in https://github.com/WordPress/performance/pull/1068.

Thanks!